cmd/show.go

```diff
@@ -35,28 +34,38 @@ var showCmd = &cobra.Command{
 		}
 		logger.SetupLogfile(logfile)
 
-		logger.Disklog.Debug("show called by the user. Pruning then listing all tunnels.")
+		pretty, err := cmd.Flags().GetBool("pretty")
+		if err != nil {
+			logger.Disklog.Warn("Problem retrieving pretty flag", err)
+		}
+
 		manager.PruneState()
 
 		pool, _ := cmd.Flags().GetString("pool")
 		id, _ := cmd.Flags().GetString("id")
 
-		logger.Disklog.Debug("Pool name searched: ", pool)
-		logger.Disklog.Debug("ID searched: ", id)
-
-		if pool == "" && id != "" {
-			output.ShowTunnelJSON(id)
-		} else if pool == "" && id == "" {
-			output.ShowStateJSON()
-		} else {
-			output.ShowPool(pool)
+		switch pretty {
+		case true:
+			output.PrettyPrint(id, pool)
+		case false:
+			if pool == "" && id != "" {
```

Comment:
Since we're shelling out the logic when pretty is set, should we do the same for this?

I can see this logic changing and then having to change the same logic in two diff files.

Reply:
https://github.com/mdsauce/sauced/issues/60
created this issue to track.  should be condensed in a future refactor/branch.  touches multiple areas around output in general.